### PR TITLE
Added compose option

### PR DESCRIPTION
**Pull request by dovry**
I created a compose file for the container, and reworded the 'Finally' section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,10 +62,28 @@ docker \
...
 ```
+Sample compose file
+-----
+```yaml
+version: "2"
+services:
+    pyload:
+      image: writl/pyload:latest
+      container_name: pyload
+      network_mode: host
```
Comment (obi12341) [inline README.md]: Why are you using network_mode: host?
I guess this is not needed and should be avoided. Can you remove it?

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+version: "2"
+services:
+    pyload:
+      image: writl/pyload:latest
+      container_name: pyload
+      network_mode: host
```
Comment (obi12341) [inline docker-compose.yml]: look comment above

**Comment by obi12341** [comment]
Thanks for your PR :) Please have a look of my review :)

**Comment by dovry** [comment]
I have modified the compose-file and readme to forward port 8000 instead of using network_mode: host :+1:

**Pull request merged**